src/main/java/ldbc/snb/datagen/util/formatter/StringDateFormatter.java

```diff
@@ -44,20 +45,14 @@ Linked Data Benchmark Council (http://www.ldbcouncil.org)
 import java.util.Date;
 
 public class StringDateFormatter implements DateFormatter {
-
-    private final ZoneId GMT = ZoneId.of("GMT");
-
-    private String formatDateTimeString = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
-    private String formatDateString = "yyyy-MM-dd";
-
     private DateTimeFormatter gmtDateTimeFormatter;
     private DateTimeFormatter gmtDateFormatter;
 
     public void initialize(LdbcConfiguration conf) {
-        formatDateTimeString = DatagenParams.getDateTimeFormat();
-        gmtDateTimeFormatter = DateTimeFormatter.ofPattern(formatDateTimeString).withZone(GMT);
-        formatDateString = DatagenParams.getDateFormat();
-        gmtDateFormatter = DateTimeFormatter.ofPattern(formatDateString).withZone(GMT);
+        String formatDateTimeString = DatagenParams.getDateTimeFormat();
+        gmtDateTimeFormatter = DateTimeFormatter.ofPattern(formatDateTimeString).withZone(DateUtils.UTC);
+        String formatDateString = DatagenParams.getDateFormat();
+        gmtDateFormatter = DateTimeFormatter.ofPattern(formatDateString).withZone(DateUtils.UTC);
```

Comment:
While `UTC` is the good choice for epochs, here we should use `GMT` to comply with the spec.
![image](https://user-images.githubusercontent.com/1402801/81499355-5167e080-92cb-11ea-93e3-6ea49c9c8db2.png)

Reply:
There's no real difference between GMT and UTC as long as we only interested in the offset (+0000), and not the geolocation of the timestamp. UTC carriers no information 'where' the timestamp was 'created', whereas using GMT (or any TZ for that matter) usually conveys that 
- it was created in a certain part of the world (countries using GMT time). 
- it was created in a certain time of the year because of DST (e.g GMT is only used in winter in the UK)
If you don't want involve these political/geographical connotations, it is better to use UTC.
What do you think?

Reply:
I'm personally in favour of UTC but then we'll need to change the spec. Which is perfectly possible but let's discuss this within the task force.

Reply:
Until we only represent the TZ offset, it the output won't change, so we don't really violate the spec.

Reply:
😌😌😌

Reply:
As this is the formatter code, It makes sense to change it back to be in line with the standard, however the rest of the code should use UTC.